torch/_subclasses/fake_tensor.py

```diff
@@ -507,6 +507,11 @@ class FakeTensor(torch.Tensor):
     fake_mode: "FakeTensorMode"
     constant: Optional[torch.Tensor]
 
+    # DEBUG_MODE on FakeTensor enables a debug mode.
+    # Today, this mode only records fake tensor creation stack traces,
+    # But in the future will add more.
+    DEBUG_MODE = False
```

Comment:
It would be nice to have a config module for this, and then have this gated by an environment variable.

We should setup some discoverability knob for this.  Maybe change the repr for FakeTensor to suggest rerunning with the envvar if you want to see the backtrace it was allocated from.

Reply:
And yes, I want a test for this too.

Reply:
I loathe to add more configs, but if you prefer, we can. Is there any config associated with Modes/dispatch/fake tensor today?

Reply:
The thing I'm looking for is, I want a discoverability error message (similar to how GraphModule prints tell you about `print_readable`), and the discoverability message has to tell me exactly what I should do to get backtraces, and the message ideally shouldn't involve editing PyTorch source code. So it's hard to imagine other ways to do it. I guess you could have an env var without a config module, but you still need one central spot to set the toggle.